proto/frequenz/api/common/v1/microgrid/electrical_components/electrical_components.proto

```diff
@@ -121,6 +121,54 @@ enum ElectricalComponentStatus {
   ELECTRICAL_COMPONENT_STATUS_INACTIVE = 2;
 }
 
+// `ElectricalComponentControlMode` explicitly defines whether an electrical
+// component is intended to be active (enabled) or inactive (disabled). This
+// intentional setting directly determines component availability to downstream
+// services and applications.
+//
+// !!! note "Control Mode & Component Availability"
+//     The Control Mode explicitly represents the user's or automation's
+//     intention to make the component either ACTIVE or INACTIVE. This directly
+//     impacts availability:
+//     - ACTIVE indicates the component is intended to be available and usable
+//       by downstream agents and services.
+//     - INACTIVE indicates the component is intended to be unavailable and
+//       ignored by downstream agents and services.
+//
+// !!! note "Relationship with Operational State"
+//     Control Mode explicitly represents intention, distinct from the
+//     component's actual internal Operational State.
+//     Operational State is independently reported by the component hardware
+//     (e.g., READY, CHARGING, DISCHARGING, ERROR).
+//
+// !!! caution "Immediate Operational Impact"
+//     Changing the Control Mode of an electrical component (e.g., via the
+//     Microgrid API) takes immediate effect on its availability to downstream
+//     agents and services. Setting a component to `INACTIVE` can thus instantly
+//     interrupt ongoing operations or cause service disruptions. Setting a
+//     component to `ACTIVE` may immediately connect it with other components,
+//     potentially causing electrical damage. Therefore, ensure changes are
+//     coordinated or planned appropriately to avoid unintended impact and/or
+//     damages.
+//
+// !!! example "Control Mode Change Impact"
+//     If a battery component's Control Mode is switched from ACTIVE to
```

Comment:
Is that only true for batteries?

Reply:
It should be true for every component that we can disable. As an example, battery works well though.

Reply:
ah its an example I missed that.